packages/axelar-wasm-std/src/msg_id/tx_hash.rs

```diff
@@ -0,0 +1,128 @@
+use core::fmt;
+use std::fmt::Display;
+use std::str::FromStr;
+
+use cosmwasm_std::HexBinary;
+use error_stack::{Report, ResultExt};
+use lazy_static::lazy_static;
+use regex::Regex;
+
+use super::Error;
+use crate::hash::Hash;
+use crate::nonempty;
+
+pub struct HexTxHash {
+    pub tx_hash: Hash,
+}
+
+impl HexTxHash {
+    pub fn tx_hash_as_hex(&self) -> nonempty::String {
+        format!("{}", HexBinary::from(self.tx_hash).to_hex())
```

Comment:
```suggestion
        HexBinary::from(self.tx_hash).to_hex()
```
`format` should be redundant here.

Reply:
Done